src/renderer/utils/createStorage.ts

```diff
@@ -47,15 +46,11 @@ export function createStorage<T>(key: string, defaultValue: T) {
 
 export function createAppStorage<T>(key: string, defaultValue: T, clusterId?: string | undefined) {
   const { logPrefix } = StorageHelper;
-  const folder = path.resolve((app || remote.app).getPath("userData"), "lens-local-storage");
+  const folder = path.resolve(getPath("userData"), "lens-local-storage");
   const fileName = `${clusterId ?? "app"}.json`;
   const filePath = path.resolve(folder, fileName);
 
-  if (!storage.initialized) {
-    init(); // called once per cluster-view
-  }
-
-  function init() {
+  if (!storage.initialized && !process.argv.includes("--integration-testing")) {
```

Comment:
ideally this `!process.argv.includes("--integration-testing")` should be in helper method that is called here

Reply:
Sounds good